### PR TITLE
Feature/add water fall chart v2

### PR DESCRIPTION
**Pull request by alejandrocabriales**
## Ticket
https://trello.com/c/qmqD8fNL/311-rc-1-reserves-chart

## Description
Add the filters

## What solved
- [X]  **(1)** Should display a dropdown with the "All MakerDao" value by default. [image.png]
- [X] **(3)** Should the dropdown display the budget options depending on the type of the finances page. [image.png]

### CODE REVIEW

File: src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx [hunk 1]
```diff
@@ -1,6 +1,10 @@
-import { styled } from '@mui/material';
+// import { styled } from '@mui/material';
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx:1]: Did you forget to remove this comment?

File: src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx [hunk 2]
```diff
@@ -9,56 +13,60 @@ interface Props {
...
+
+      <Title isLight={isLight}>{title}</Title>
+    </Container>
+  );
+};
 
 export default LegendItemChart;
 
-const Container = styled('div')(({ theme }) => ({
+const Container = styled.div<WithIsLight>({
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx:31]: `WithIsLight` is not used here so it should not be necessary

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx]: you are right, done

File: src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/LegendItemChart/LegendItemChart.tsx]: done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**